spec/course_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/course'
+require './lib/student'
+
+RSpec.describe Course do
+  
+  it 'can initialize' do
+    course = Course.new("Calculus", 2)    
+    expect(course).to be_instance_of(Course)
+    expect(course.name).to eq("Calculus")
+    expect(course.capacity).to eq(2)
+    expect(course.students).to eq([])
+  end
+  it 'course can enroll student and be full if they exceed capacity' do
+    course = Course.new("Calculus", 2)    
+    expect(course.full).to eq(false)
```

Comment:
`full` is not a method so this is failing because the program can't find a method called `full` in your `Course` class. The question mark does make it a wholly different method when the program is looking for it.
```suggestion
    expect(course.full?).to eq(false)
```